react/components/molecules/key-values/key-values.js

```diff
@@ -34,4 +78,15 @@ export class KeyValues extends PureComponent {
     }
 }
 
+const styles = StyleSheet.create({
+    keyValuesColumns: {
+        flexDirection: "row",
+        flexWrap: "wrap",
+        justifyContent: "space-between"
+    },
+    keyValueTwoColumns: {
+        width: "48%"

```

Comment:
Gotcha 😅. Why not 50%?

Reply:
To have some space between them. I use this in combination with ` justifyContent: "space-between"`.

Margin doesnt count for the component width, so the flexWrap doesnt work properly.
When using padding, the border doesn't shift, stays the same

This was the solution that seemed ok.

Reply:
Ok I see. As a last check, can you please confirm how is it currently working on the order activity list? There we use `FlatList` but maybe you can take inspiration from there, just to confirm if there's an alternative.